Tools/wasm/wasi.py

```diff
@@ -26,6 +26,8 @@
 LOCAL_SETUP = CHECKOUT / "Modules" / "Setup.local"
 LOCAL_SETUP_MARKER = "# Generated by Tools/wasm/wasi.py\n".encode("utf-8")
 
+WASMTIME_VAR = "{WASMTIME}"
```

Comment:
I found this whole thing with the extra curly braces a bit confusing.  IIUC:

- `WASMTIME` is the name of an envvar that should point to the `wasmtime` binary when all is said and done
- `{WASMTIME}` is a special token in the default `host_runner` that is replaced with the value of `WASMTIME`.

Therefore, I think it would be more obvious to do:

```suggestion
WASMTIME_VAR = "WASMTIME"
WASMTIME_VAR_TOKEN = f"{{{WASMTIME}}}"
```

and then update / simplify the uses of these variables below.

Reply:
Maybe `WASMTIME_VAR_TOKEN = "{%s}" % WASMTIME`? :)

Reply:
I think you meant for it to be `WASMTIME_VAR_TOKEN = f"{{{WASMTIME_VAR}}}", but I get your point.